src/components/MultiHopTrade/components/TradeInput/components/Claim/hooks/useArbitrumClaimsByStatus.tsx

```diff
@@ -36,47 +34,40 @@ export type ClaimDetails = Omit<ClaimStatusResult, 'status'> & {
 
 type ClaimsByStatus = Record<ClaimStatus, ClaimDetails[]>
 
-export const useArbitrumClaimsByStatus = (txs: Tx[]) => {
+export const useArbitrumClaimsByStatus = (props?: { isDisabled?: boolean }) => {
   const translate = useTranslate()
 
   const ethAsset = useAppSelector(state => selectAssetById(state, ethAssetId))
+  const arbitrumWithdrawTxs = useAppSelector(selectArbitrumWithdrawTxs)
 
   const l1Provider = getEthersV5Provider(KnownChainIds.EthereumMainnet)
   const l2Provider = getEthersV5Provider(KnownChainIds.ArbitrumMainnet)
 
   const queries = useMemo(() => {
     return {
-      queries: txs.map(tx => {
+      queries: arbitrumWithdrawTxs.map(tx => {
         return {
           queryKey: ['claimStatus', { txid: tx.txid }],
           queryFn: async () => {
             const receipt = await l2Provider.getTransactionReceipt(tx.txid)
             const l2Receipt = new ChildTransactionReceipt(receipt)
             const events = l2Receipt.getChildToParentEvents()
             const messages = await l2Receipt.getChildToParentMessages(l1Provider)
-
             const event = events[0]
             const message = messages[0]
-
             const status = await message.status(l2Provider)
             const block = (await message.getFirstExecutableBlock(l2Provider))?.toNumber()
-
```

Comment:
no start imports OR whitespace for you @woodenfurniture?? :rofl:

Reply:
argh lol this must've happened when I reverted some prior changes